python-use-input-in-inline-script.yaml

```diff
@@ -0,0 +1,50 @@
+id: python-use-input-in-inline-script
+namespace: company.team
+
+inputs:
+  - id: pokemon
+    type: STRING
+    defaults: pikachu
+
+  - id: your_age
+    type: INT
+    defaults: 25
+
+tasks:
+  - id: inline_script
+    type: io.kestra.plugin.scripts.python.Script
+    description: Fetch the pokemon detail and compare its experience
+    taskRunner:
+      type: io.kestra.plugin.scripts.runner.docker.Docker
+    containerImage: ghcr.io/kestra-io/pydata:latest
+    script: |
+      import requests
+      import json
+
+      url = "https://pokeapi.co/api/v2/pokemon/{{ inputs.pokemon }}"
+      response = requests.get(url)
+
+      if response.status_code == 200:
+          pokemon = json.loads(response.text)
+          print(f"Base experience of {{ inputs.pokemon }} is { pokemon.get('base_experience') }")
+          if pokemon.get('base_experience') > int("{{ inputs.your_age }}"):
+              print("{{ inputs.pokemon }} has more base experience than your age")
+          else:
+              print("{{ inputs.pokemon}} is too young!")
+      else:
+          print(f"Failed to retrieve the webpage. Status code: {response.status_code}")
+
+extend:
+  title: Create a Python inline script that takes input using an expression
+  description: >-
```

Comment:
Can you add to the description about the task runner is defined but this is the default value anyway. Also explain that everything in containers